library/core/src/num/dec2flt/algorithm.rs

```diff
@@ -64,8 +64,7 @@ mod fpu_precision {
             asm!(
                 "fldcw ({})",
                 in(reg) &cw,
-                // FIXME: We are using ATT syntax to support LLVM 8 and LLVM 9.
-                options(att_syntax, nostack),
+                options(nostack),
```

Comment:
Hm, not super sure, but I think these want to become `fldcw [{}]` and `fnstcw [{}]` now.